client/.gitignore

```diff
@@ -13,6 +13,7 @@
 
 # misc
 .DS_Store
+!.env
```

Comment:
In React code, you want to push `.env` into GitHub because it contains values that are shared across developers and visible to customers. But with our deployment solution, you can't access the database using this approach, anyway. You have to just use `/api/...` and rely on the "proxy" field in package.json.